lib/bot.js

```diff
@@ -36,6 +36,8 @@ class Bot {
     this.channels = _.values(options.channelMapping);
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
+    this.ignoreBots = options.ignoreBots;
+    this.ignoreIrcNicks = options.ignoreIrcNicks || [];
```

Comment:
I'm unclear where such a circumstance would arise? I agree that if such a situation did occur then user ID would be better, but don't expect it'd occur enough that it'd outweigh the benefit from using `username#discriminator`.

I mean, not that there's *much* difference between the two, so I'm mostly of the opinion "pick at least one of these" here.